### PR TITLE
add pypalettes .yml file

### PR DESCRIPTION
**Pull request by JosephBARBIERDARNAL**
Hello!

Coming from [this tweet](https://x.com/matplotlib/status/1805620420927709227) for adding the [pypalettes](https://github.com/JosephBARBIERDARNAL/pypalettes) third party.

In this PR, I created pypalettes.yml, with:

```yml
name: pypalettes
repo: JosephBARBIERDARNAL/pypalettes
section: colormaps and styles
description: Large collection of colormaps.
site: https://python-graph-gallery.com/color-palette-finder/
keywords: [colormaps, styles]
```

do I need to do something else?

### CODE REVIEW

**Comment by JosephBARBIERDARNAL** [comment]
forgot to create a new branch, hope that's not an issue

**Comment by QuLogic** [comment]
> forgot to create a new branch, hope that's not an issue

Not really a problem, unless you want to submit more than one PR.

> do I need to do something else?

You only need to fix the linting error.

File: packages/pypalettes.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+name: pypalettes
+repo: JosephBARBIERDARNAL/pypalettes
+section: colormaps and styles
+description: Large collection of colormaps.
```
[the remark this answers is not in the document]
Comment (JosephBARBIERDARNAL) [reply packages/pypalettes.yml:4]: makes sense, thanks!

**Comment by JosephBARBIERDARNAL** [comment]
> You only need to fix the linting error.

Not sure how to do that. I did

`pip intall pre-commit`
`pre-commit install`

commit+push, but does not seem to change the linting issue? did I miss something?

**Pull request merged**